workers/autopruneworker.py

```diff
@@ -95,5 +161,6 @@ def create_gunicorn_worker():
             time.sleep(100000)
 
     logging.config.fileConfig(logfile_path(debug=False), disable_existing_loggers=False)
+    GlobalLock.configure(app.config)
```

Comment:
How about configuring the lock if default auto prune policy exists (around [here](https://github.com/quay/quay/pull/2950/files#diff-7db0e9f1060956a8ddbfed7876897f57385740a3a58742640f2761fa902dde35R51))?

Reply:
add a conditional for creating the lock